modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -278,7 +278,8 @@ def parsed_response(response, action:, key:, transform:)
       end
     end
 
-    def make_request(endpoint:, action:, body:, key: nil, namespaces: {}, transform_response: true) # rubocop:disable Metrics/MethodLength, Metrics/ParameterLists
+    # rubocop:disable Metrics/ParameterLists
```

Comment:
I don't think the changes in this file are necessary

Reply:
Rachal Cassity asked us to put those in as they were in the code before our changes and got removed in the process of all the changes done throughout this PR.  They exist currently in the master branch.